docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/utility-scripts.md

```diff
@@ -20,9 +21,8 @@
    ![](../images/creds-mac.png)
 
 ### Assign MAC Addresses to existing credentials
-> **Note:** Currently EdgeX is unable to provide a way to query the names of existing secrets from the secret store, so this method
-> only works with credentials which have a key in the CredentialsMap. If the credentials were added via these
-> utility scripts, a placeholder key was added for you to the CredentialsMap.
+!!! Note
+      Currently EdgeX is unable to provide a way to query the names of existing secrets from the secret store, so this method only works with credentials which have a key in the CredentialsMap. If the credentials were added via these utility scripts, a placeholder key was added for you to the CredentialsMap.
```

Comment:
```suggestion
!!! note
    Currently EdgeX is unable to provide a way to externally query the names of existing secrets from the secret store, so this method only works with credentials which have a key in the CredentialsMap. If the credentials were added via these utility scripts, a placeholder key was added for you to the CredentialsMap.
```

Reply:
Is this note applicable to both Secure and Non-Secure modes?

I think it would be useful for this note to have a link to the `Credentials` page.